static/include/components/apis/analogreader.md

```diff
@@ -1,4 +1,4 @@
 | Method Name                         | Description                                                             |
 | ----------------------------------- | ----------------------------------------------------------------------- |
-| [`Read`](/components/board/#read)   | Read the current integer value of the digital signal output by the ADC. |
+| [`ReadAnalogReader`](/components/board/#readanalogreader)   | Read the current integer value of the digital signal output by the ADC. |
```

Comment:
This is not a fucntion anywhere in any of our sdks.

Reply:
@npentrel is it ok to change / if proto doesn't match here?

Reply:
I think so @andf-viam tagging you for awareness re: automation